### PR TITLE
Update integration README by adding a missing command-line flag and fixing typos

### PR DESCRIPTION
**Pull request by Winbobob**
**Please provide a description of this PR:**
Update integration README by adding a missing command-line flag `-istio.test.skip` and fixing typos.

### CODE REVIEW

**Comment by ericvn** [comment]
I wonder if the flags might be better in a table: Name, type, description.

Looking at the README, some of the descriptive text goes off the right hand side of the page and is lost. A table would allow all the text to be seen.

File: tests/integration/README.md [hunk 1]
```diff
@@ -514,61 +514,27 @@ pass command-line flags to the test while running under the debugger, you can us
...
-  -istio.test.skipVM bool
-        Skip all the VM related parts in all the tests. (default is "false")
-
-  -istio.test.helmRepo string
-        Overwrite the default helm Repo used for the tests.
-
-  -istio.test.ambient bool
-        Indicate the use of ambient mesh.
-```
+| Name | Type | Description |
```
Comment (bleggett) [inline tests/integration/README.md:517]:
```suggestion
| Name | Type | Description | Default |
```

might be good.

+1 overall

**Review by bleggett — APPROVED** [review]
+1 and ty

**Review by keithmattix — APPROVED** [review]
Approving pending other feedback

**Review by jacob-delgado — APPROVED** [review]

**Pull request merged**